etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/metadata/partner/PartnerMetadataOrchestrator.java

```diff
@@ -111,21 +93,13 @@ public void updateMetadataForReceivedMessage(
                 "Updating metadata with sender: {}, timeReceived: {} and hash",
                 sender,
                 timeReceived);
-        PartnerMetadata partnerMetadata =
-                new PartnerMetadata(
-                        receivedSubmissionId,
-                        sender,
-                        timeReceived,
-                        null,
-                        messageHash,
-                        PartnerMetadataStatus.PENDING,
-                        messageType,
-                        sendingApplicationDetails,
-                        sendingFacilityDetails,
-                        receivingApplicationDetails,
-                        receivingFacilityDetails,
-                        placerOrderNumber);
-        partnerMetadataStorage.saveMetadata(partnerMetadata);
+        PartnerMetadata updatedPartnerMetadata =
+                partnerMetadata
+                        .withDeliveryStatus(PartnerMetadataStatus.PENDING)
+                        .withSender(sender)
+                        .withTimeReceived(timeReceived)
+                        .withTimeDelivered(null);
```

Comment:
Wasn't it already created with `PENDING` in the usecase, so I don't think we need to set this here.  And the time delivered doesn't need to be overwritten either I don't think.  What was your reasoning?

Reply:
I actually didn't work on these partner metadata changes but I'll check with the authors

Reply:
I checked and it's actually not created with PENDING status in the usecase. It's created with null deliveryStatus

Reply:
I think if it is `null`, then it will default `PENDING`.  See around line 36.

Reply:
Hmmm good point, I missed that. I'll remove it then

Reply:
Done